src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: left 19px top 32px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__title {
+    margin-top: 206px;
+    margin-bottom: 4px;
+    font-family: $main-font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $main-text-color;
```

Comment:
Ensure that the variable `$main-text-color` is defined in your SCSS files. If not, define it to avoid compilation errors.